tasks/create.yaml

```diff
@@ -14,8 +14,14 @@ tasks:
       architecture:
         description: The architecture of the package to create
         default: ${UDS_ARCH}
+      config:
+        description: Optionally supply a path to the desired Zarf Configuration for the package
+        default: ""
+        required: false
     actions:
-      - cmd: |
+      - env:
+          - ZARF_CONFIG=${{ .inputs.config }}
```

Comment:
Is LFAI standardized enough to match the UDS Config when dealing with bundles where it also takes `path`? `UDS_CONFIG=${{ .inputs.path }}/${{ .inputs.config }}`

Reply:
I standardized the Zarf deploy/create tasks with the UDS ones. Both now use the original path and the config filename. Both declare the `[ZARF | UDS]_CONFIG` via the `env` field instead of in-line of the `cmd`. How does that look to you now?